src/test/configure-feature.js

```diff
@@ -2,198 +2,117 @@ import { describe } from 'riteway';
 import dom from 'cheerio';
 import React from 'react';
 import ReactDOMServer from 'react-dom/server';
-import withContext from '../test-fixtures/with-context';
-import configureFeature from '../configure-feature';
 
-const render = ReactDOMServer.renderToStaticMarkup;
+import { configureFeature } from '../configure-feature';
+import { Provider } from '../context';
 
-const createTestComponent = componentName => ({ propCheck } = {}) => (
-  <div>
-    <div className={componentName} />
-    <div className="prop-check">{propCheck}</div>
-  </div>
+const render = ReactDOMServer.renderToStaticMarkup;
+const createTestComponent = componentName => () => (
+  <div className={componentName} />
 );
 
-describe('configureFeature(Default)(FeatureName)(Feature, Fallback)', async should => {
+describe('configureFeature(Inactive)(FeatureName)(Feature)', async should => {
   const { assert } = should();
 
-  const Default = createTestComponent('default');
-  const Feature = createTestComponent('feature');
-  const Fallback = createTestComponent('fallback');
+  const ActiveComponent = createTestComponent('active');
+  const InactiveComponent = createTestComponent('inactive');
 
-  const ConfiguredFeature = configureFeature(Default)('game')(
-    Feature,
-    Fallback
+  const ConfiguredFeature = configureFeature(InactiveComponent)('game')(
+    ActiveComponent
   );
 
-  const FeatureWithContext = withContext([])(ConfiguredFeature);
-
-  const propCheck = 'bacon and eggs';
-
-  const $ = dom.load(render(<FeatureWithContext propCheck={propCheck} />));
+  const features = [];
 
-  assert({
-    given: 'the feature is not enabled and there is a Fallback component',
-    should: 'not render the Default component',
-    actual: $('.default').length,
-    expected: 0
-  });
+  const $ = dom.load(
+    render(
+      <Provider value={features}>
+        <ConfiguredFeature />
+      </Provider>
+    )
+  );
 
   assert({
-    given: 'the feature is not enabled and there is a Fallback component',
-    should: 'not render the Feature component',
-    actual: $('.feature').length,
+    given: 'the feature is not enabled and there is a Inactive component',
+    should: 'not render the Active component',
+    actual: $('.active').length,
     expected: 0
   });
 
   assert({
-    given: 'the feature is not enabled and there is a Fallback component',
-    should: 'render the Fallback component',
-    actual: $('.fallback').length,
+    given: 'the feature is not enabled and there is a Inactive component',
+    should: 'render the Inactive component',
+    actual: $('.inactive').length,
     expected: 1
   });
-
-  assert({
-    given: 'props',
-    should: 'pass through props',
-    actual: $('.prop-check').text(),
-    expected: propCheck
-  });
 });
 
-describe('configureFeature(Default)(FeatureName)(Feature, Fallback)', async should => {
+describe('configureFeature(Inactive)(FeatureName)(Feature)', async should => {
```

Comment:
please use block scope instead of repeating the same `describe` again, see the Feature test file for an example. This is just a preference I am starting to like and would like to keep it consistent. Also, adjust argument names to match the signature here as well please.